.github/dependabot.yml

```diff
@@ -16,3 +21,5 @@ updates:
     directory: "/"
     schedule:
       interval: "weekly"
+    ignore:
+      - dependency-name: "hashicorp/aws"
```

Comment:
Does this cover `./dns` and `./security_groups` as well?

Reply:
🦅👀 It does not. I didn't realize those directories were missing configurations when I was churning through these. Please see [99fa12f](https://github.com/cisagov/cool-sharedservices-freeipa/pull/49/commits/99fa12f53287a8983adab13ecb4aaabdf9059259) for those getting Dependabot configurations.

Reply:
Not really :eagle: :eyes: - I just happened to be up to my elbows in this code when I reviewed the PR.

Thanks for adding the configurations for those two directories!